### PR TITLE
Aroosi branch

### PR DESCRIPTION
**Pull request by aroosil**

### CODE REVIEW

File: src/commons/shuffle/button-icon.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import {inject, customElement, bindable} from 'aurelia-framework';
+import {EventAggregator} from 'aurelia-event-aggregator';
+import {autoinject} from "aurelia-dependency-injection";
+
+@inject(EventAggregator)
+@inject(Element)
+@customElement('button-icon')
+export class ButtonIcon {
+  @bindable elemName = '';
+  stringArray = ['crm', 'accounting', 'calendar', 'contacts', 'manufacturing'];
```
Comment (daysleeperx) [inline src/commons/shuffle/button-icon.js:10]: is this array used at all?

File: src/commons/shuffle/button-icon.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import {inject, customElement, bindable} from 'aurelia-framework';
+import {EventAggregator} from 'aurelia-event-aggregator';
+import {autoinject} from "aurelia-dependency-injection";
+
+@inject(EventAggregator)
+@inject(Element)
```
Comment (daysleeperx) [inline src/commons/shuffle/button-icon.js:6]: can injects be combined or smth?

File: src/commons/shuffle/button-icon.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  namething = '';
+
+
+
+  constructor() {
+    this.element = {
+          active: ''
+        };
+  }
+  x;
```
Comment (daysleeperx) [inline src/commons/shuffle/button-icon.js:24]: x;? :)

File: src/commons/workspaces-menu/workspace-icon.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import {inject, customElement, bindable} from 'aurelia-framework';
+import {EventAggregator} from 'aurelia-event-aggregator';
+import {autoinject} from "aurelia-dependency-injection";
+
+@inject(EventAggregator)
+@inject(Element)
```
Comment (daysleeperx) [inline src/commons/workspaces-menu/workspace-icon.js:6]: same here regarding injects

**Comment by daysleeperx** [comment]
general comment: can branch name be more feature related, not uniid? :)

File: src/commons/shuffle/button-icon.js (hunk 1 above)
Comment (aroosil) [reply src/commons/shuffle/button-icon.js:10]: not used

File: src/commons/shuffle/button-icon.js (hunk 3 above)
Comment (aroosil) [reply src/commons/shuffle/button-icon.js:24]: oih :D

File: src/commons/workspaces-menu/workspace-icon.js (hunk 4 above)
Comment (aroosil) [reply src/commons/workspaces-menu/workspace-icon.js:6]: workspace icon still not ready yet, just random code there

**Pull request merged**